.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Description
+<!--- Describe your problem as fully as you can. -->
+
+## Steps to reproduce
+<!-- What do you do that leads to the problem you're seeing? -->
+
+## Your environment
```

Comment:
I'm in favor of asking for their environment by default, but everything else seems onerous. 

For example, if someone is requesting a new feature, they're forced to write it in a way that conforms to our guidelines. Technically new features don't have steps to reproduce, but often times I've seen people try to describe their new features in this format and end up describing what they want in a poor manner.